test/test_zache.rb

```diff
@@ -83,4 +83,22 @@ def test_remove_key_with_sync_false
     assert(cache.exists?(:hey) == false)
     assert(cache.exists?(:wey) == true)
   end
+
+  def test_remove_all_with_sync
+    cache = Zache.new
```

Comment:
@Qew7 we need a stronger test here. Try to use `Threads` and do `get()` and `remove_all` at the same time in multiple threads. See what happens.